README.md

```diff
@@ -20,7 +20,7 @@
 
 ## How do I update the data for a library?
 
-- Find the entry for the library in [`react-native-libraries.json`](https://github.com/react-native-community/directory/blob/main/react-native-libraries.json) file. (Not there? [Add it the library](#how-do-i-add-a-library).)
+- Find the entry for the library in [`react-native-libraries.json`](https://github.com/react-native-community/directory/blob/main/react-native-libraries.json) file. (Not there? [Add it to the library](#how-do-i-add-a-library).)
```

Comment:
Nice catch! 👍 

Let's also change wording a bit, to better differentiate the package/library form the directory/registry in the sentence:

```suggestion
- Find the entry for the library in [`react-native-libraries.json`](https://github.com/react-native-community/directory/blob/main/react-native-libraries.json) file. (Not there? [Add it to the directory](#how-do-i-add-a-library).)
```